client/src/TenderClient.js

```diff
@@ -21,7 +21,9 @@ const handleAuthError = (error) => {
 	if (error.response && error.response.status === 401) {
 		localStorage.removeItem("authToken");
 		localStorage.removeItem("userType");
-		window.location.href = "/";
+		if (window.location.href !== "/") {
+			window.location.href !== "/";
+		}
```

Comment:
This is incorrect. Please test your code before sending it for review

Reply:
i did test it and it was working